Bottom: cf778b78b35c23cde722678972976d10047c1c06
Top:    6c081c25aabbcbf36a2b3fd4ad7fef85130c699b
Author: Mark de Wever <koraq@xs4all.nl>
Date:   2021-08-14 15:42:07 +0200

Refresh of playing-with-json

---

diff --git a/cmake/CodeCoverage.cmake b/cmake/CodeCoverage.cmake
index 839c8a8..e3437ec 100644
--- a/cmake/CodeCoverage.cmake
+++ b/cmake/CodeCoverage.cmake
@@ -204,6 +204,7 @@ endif()
 # ccov-show-${TARGET_NAME} : Prints to command line detailed per-line coverage information.
 # ccov-all : Generates HTML code coverage report, merging every target added with 'ALL' parameter into a single detailed report.
 # ccov-all-report : Prints summary per-file coverage information for every target added with ALL' parameter to the command line.
+# ccov-all-export : Exports the coverage report to a JSON file.
 #
 # Required:
 # TARGET_NAME - Name of the target to generate code coverage for.
